### ISSUE TITLE
Commits Views not working

### ISSUE THREAD

**Issue by nunojsa** [gitkraken/vscode-gitlens#3019]
### Description

I'm trying to see the commits view in linux kernel and it just hangs and does not show a thing. I'm not totally sure this is directly related to gitlens but it is definitely affecting it... Logs below...

I'm also not sure if this might be related with another issue with commits graph that's also not working as expected. Very low number of commits shown.

### GitLens Version

14.5.0

### VS Code Version

Version: 1.84.2
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:50:47.800Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 6.5.10-200.fc38.x86_64

### Git Version

git version 2.41.0

### Logs, Screenshots, Screen Captures, etc

```
[2023-11-15 09:51:56.078] [GIT  ] [/home/nsa/work/linux-iio] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cn%x3e%x20%cN%n%x3cm%x3e%x20%cE%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e -M -m --name-status --full-history dev/iio-backend -- [*30305ms] (slow)
[2023-11-15 09:51:56.080] /home/nsa/.vscode/extensions/eamodio.gitlens-2023.11.1504/dist/gitlens.js:1:221222
Error: Command output exceeded the allocated stdout buffer. Set 'options.maxBuffer' to a larger value than 104857600 bytes
[2023-11-15 09:52:27.142] [GIT  ] [/home/nsa/work/linux-iio] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cn%x3e%x20%cN%n%x3cm%x3e%x20%cE%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e -M -m --name-status --full-history dev/iio-backend -- [*31051ms] (slow)
[2023-11-15 09:52:27.145] /home/nsa/.vscode/extensions/eamodio.gitlens-2023.11.1504/dist/gitlens.js:1:221222
Error: Command output exceeded the allocated stdout buffer. Set 'options.maxBuffer' to a larger value than 104857600 bytes
```

**Comment by eamodio** [comment]
Can you please see if this issue is still reproducible in the latest pre-release edition of GitLens? Thanks!

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![image](https://user-images.githubusercontent.com/641685/228608740-6071bc19-7b1a-44c8-843f-d2af44d6114b.png)

**Comment by nunojsa** [comment]
> Can you please see if this issue is still reproducible in the latest pre-release edition of GitLens? Thanks!

Yes, still an issue in the pre-release...

**Comment by eamodio** [comment]
Can you please provide a full set of logs? Also I see the Git `log` command run doesn't have a count specified and I'm not sure how that is happening. Can you also show what you are doing/seeing in the UI? Thanks.

Please enable debug logging by running the _GitLens: Enable Debug Logging_ command from the Command Palette (<kbd>F1</kbd> or <kbd>ctrl/cmd</kbd>+<kbd>shift</kbd>+<kbd>p</kbd>). This will enable logging to the _GitLens_ & _GitLens (Git)_ channels in the _Output_ pane.

Once enabled, please reproduce the issue, and attach the logs from the _GitLens_ and _GitLens (Git)_ channels.

Thank you!

**Comment by eamodio** [comment]
Can you please verify this fix in tomorrow's (`v2023.11.08xx`) pre-release edition of GitLens? Thank you so much!

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![image](https://user-images.githubusercontent.com/641685/228608740-6071bc19-7b1a-44c8-843f-d2af44d6114b.png)
